src/content/pricing/config.ts

```diff
@@ -102,7 +103,7 @@ export const AiAgentsPricingInfo: Props[] = [
     ],
     cta: 'Start free trial',
     variant: 'primary-white',
-    url: `${dashboardUrl}/agents`,
+    url: `${xyzWebsiteUrl}?referral=agents`,
```

Comment:
I'm under the impression that the original URL is correct. That is the URL you want to redirect the user if the user is already authenticated. Which is /agents. Correct? 

```suggestion
    url: `${dashboardUrl}/agents`,
```